### PR TITLE
chore(rebeam): update pres def to be more realistic

### PR DESCRIPTION
**Pull request by jrhender**

### CODE REVIEW

**Comment by Harasz** [comment]
My question is, do we need this? Issuer fields are required by a role definition.

**Comment by Harasz** [comment]
Okay, I figured it out. Check tests in this PR https://github.com/energywebfoundation/ssi/pull/42

File: apps/nestjs-wallet/test/vc-api/exchanges/rebeam/rebeam-cpo-node.ts [hunk 1]
```diff
@@ -34,10 +34,17 @@ export class RebeamCpoNode {
...
                             type: 'string',
-                            const: 'customer.roles.rebeam.apps.elia.iam.ewc'
+                            const: 'customer.roles.rebeam.apps.eliagroup.iam.ewc'
+                          }
+                        },
+                        {
+                          path: ['$.credentialSubject.issuerFields[*]'],
+                          filter: {
+                            type: 'object'
+                            // TODO: how can we ask for { "key": "accountId", "value": <any string> }
```
Comment (Harasz) [inline apps/nestjs-wallet/test/vc-api/exchanges/rebeam/rebeam-cpo-node.ts:47]:
```suggestion
                          path: ['$.credentialSubject.issuerFields[*].key'],
                          filter: {
                          path: ['$.credentialSubject.issuerFields[*].key'],
                          filter: {
                            type: 'string',
                            const: 'accountId'
                          }
```

Comment (jrhender) [reply apps/nestjs-wallet/test/vc-api/exchanges/rebeam/rebeam-cpo-node.ts:47]: Hmm, yea, maybe it's that easy!

**Review by Harasz — APPROVED** [review]

**Pull request merged**